src/components/cards/ListViewTaskCard.tsx

```diff
@@ -43,15 +43,17 @@ export const ListViewTaskCard = ({
         className="task-list-card"
         sx={{
           userSelect: 'none',
+          cursor: 'default',
+          borderBottom: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
           ':hover': {
-            bgcolor: (theme) => theme.color.gray[100],
+            bgcolor: (theme) => alpha(theme.color.gray[100], 0.5),
```

Comment:
this is still being rendered as `#F8F9FB` where as the color in Figma is `#FBFCFD`.